notification/app/notification/NotificationApplicationLoader.scala

```diff
@@ -114,6 +108,55 @@ class NotificationApplicationComponents(context: Context) extends BuiltInCompone
     new FrontendAlerts(frontendConfig, wsClient)
   }
 
+  lazy val topicRegistrationCounter: TopicRegistrationCounter = new ReportTopicRegistrationCounter(
+    wsClient,
+    configuration.get[String]("report.url"),
+    configuration.get[String]("report.apiKey")
+  )
+
+  lazy val sqsClient: AmazonSQSAsync = AmazonSQSAsyncClientBuilder.standard()
+    .withCredentials(credentialsProvider)
+    .withRegion(EU_WEST_1)
+    .build()
+
+  lazy val guardianIosNotificationSender: GuardianNotificationSender = new GuardianNotificationSender(
+    sqsClient = sqsClient,
+    registrationCounter = topicRegistrationCounter,
+    platform = iOS,
+    sqsUrl = configuration.get[String]("notifications.queues.ios")
+  )
+
+  lazy val guardianAndroidNotificationSender: GuardianNotificationSender = new GuardianNotificationSender(
+    sqsClient = sqsClient,
+    registrationCounter = topicRegistrationCounter,
+    platform = Android,
+    sqsUrl = configuration.get[String]("notifications.queues.android")
+  )
+
+  lazy val guardianNewsstandNotificationSender: GuardianNotificationSender = new GuardianNotificationSender(
+    sqsClient = sqsClient,
+    registrationCounter = topicRegistrationCounter,
+    platform = Newsstand,
+    sqsUrl = configuration.get[String]("notifications.queues.ios")
+  )
+
+  def withFilter(notificationSender: NotificationSender, invertCondition: Boolean): NotificationSender =
+    new FilteredNotificationSender(notificationSender, topicRegistrationCounter, invertCondition)
+
+  lazy val notificationSenders = List(
+    withFilter(guardianIosNotificationSender, invertCondition = false),
+    withFilter(guardianAndroidNotificationSender, invertCondition = false),
+    withFilter(guardianNewsstandNotificationSender, invertCondition = false),
+    gcmNotificationSender,
+    apnsNotificationSender,
+    newsstandShardNotificationSender,
```

Comment:
ok so if I understand this means notification will be pass to all senders? right

Reply:
Yes, you're correct, but we filter notifications sent on the SQS for now